src/components/Images/MyGallery.vue

```diff
@@ -24,35 +26,56 @@ const filteredSessions = computed(() => {
   return filtered
 })
 
+const totalPages = computed(() => {
+  return Math.ceil(filteredSessions.value.length / pageSize)
+})
+
 const getThumbnails = async (observationId) => {
   await fetchApiCall({
-    url: configurationStore.thumbnailArchiveUrl + `thumbnails/?observation_id=${observationId}&size=large`,
+    url: `${configurationStore.thumbnailArchiveUrl}thumbnails/?observation_id=${observationId}&size=large`,
```

Comment:
Maybe I'm missing something, but if these thumbnails are for the page where it lists your sessions and all their little thumbnails under the dates then I think it should use the small thumbnails, not the large ones. The large ones are like 1/2 your screen size so I would save those for like opening a single image into a larger view...

Reply:
okey dokes! I should really start testing on the ipad

Reply:
and to respond to your top comment, I can definitely add the `limit` query param to the call. But I'm not sure what the solution could be. Maybe we talk about this in zoom